### ISSUE TITLE
Feature request: Configurable format for "datetime" editor

### ISSUE THREAD

**Issue by shtabco** [json-editor/json-editor#1087]
### General information

- json-editor version: 2.6.1

### Expected behavior

It would be great to have the ability to customize datetime format. If there is a way of doing that, can you please describe it?

### Actual behavior

It seems that datetime editor uses format "2022-01-07T11:19" by default and doesn't have a way to change it to "2022-01-07T11:19:10Z" for example.

**Comment by schmunk42** [comment]
Is this covered by the spec?

https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times
https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

**Comment by shtabco** [comment]
Yes,  https://datatracker.ietf.org/doc/html/rfc3339#section-5.6 covers the requirement. As I understand "date-time" format of JSON schema represents it. Looks like "data-time" type doesn't have datetime picker editor in "json-editor". Is there any way to achieve in current implementation?
